### PR TITLE
Return key behavior in Comment revised, issue#143.

### PR DESCRIPTION
**Pull request by ghost**
When add new comment in the Calendar events,  a buttom is added to submit comment, instead of ENTER key.

### CODE REVIEW

File: app/components/Thread/index.jsx [hunk 1]
```diff
@@ -11,6 +11,8 @@ const mui = require('material-ui');
 const md = require('../SmartEditor/markdown');
 const LoginStore = require('../../stores/LoginStore');
 
+require('./style.less');
```
Comment (LeonLiuY) [inline app/components/Thread/index.jsx]: What's the purpose of this line?

Comment (ghost) [inline app/components/Thread/index.jsx]: I was tring to modify the scroll behaviour of the comments list, but gave up finally and forgot to remove this line of code. [When more comments exists, the input element will be pushed down outside the screen].

Comment (LeonLiuY) [inline app/components/Thread/index.jsx]: In that case I suggest you'd better create a new issue, describe the symptom, and make a new branch to solve it.

Comment (LeonLiuY) [inline app/components/Thread/index.jsx]: Remember the topic of this issue is `Return key behavior in Comment`

**Pull request merged**